src/katsdpk8spoc/sdpcontroller.py

```diff
@@ -360,8 +488,10 @@ def main():
     swagger.add_routes(
         [
             web.get("/", home_page),
-            web.get("/start", start_handle),
-            web.get("/stop", stop_handle),
+            web.post("/product-configure", product_configure_handle),
+            web.get("/capture-init", capture_init_handle),
+            web.get("/capture-done", capture_done_handle),
```

Comment:
would POST action not make this more consistent. Lets make the capture-* and deconfigure POST then its the same as product-configure.

Reply:
done